src/routes/frog/+page.svelte.md

```diff
@@ -1,3 +1,8 @@
+<script lang="ts">
+let value = Math.floor(Math.random() * 999) + 1;
+</script>
+
 # You have been frogged!
+
 ![gay frog is not real and cant hurt you. gay frog:](https://github.com/Pridecraft-Studios/website/blob/main/static/assets/img/frog/frog.png?raw=true)
```

Comment:
This is not how to embed links; use `static` as if it was root; `/assets/img/frog/frog.png`